app/components/SourceSelector.vue

```diff
@@ -1,99 +1,69 @@
+  <div class="source-selector">
+    <div class="studio-controls-top">
+      <h4 class="studio-controls__label" v-tooltip.bottom="layersTooltip">
+        Sources
+      </h4>
+      <div>
+        <i
+          class="icon-add-folder icon-btn icon-btn--lg"
+          @click="addFolder"
+          v-tooltip.bottom="addGroupTooltip" />
+        <i
+          class="icon-add icon-btn icon-btn--lg"
+          @click="addSource"
+          v-tooltip.bottom="addLayerTooltip" />
+        <i
+          class="icon-subtract icon-btn icon-btn--lg"
+          :class="{ disabled: activeItemIds.length === 0}" @click="removeItems"
+          v-tooltip.bottom="removeLayersTooltip" />
+        <i
+          :class="{ disabled: !canShowProperties()}"
+          class="icon-settings icon-btn"
+          @click="sourceProperties"
+          v-tooltip.bottom="openLayerPropertiesTooltip" />
+      </div>
     </div>
-  </div>
 
-  <sl-vue-tree
-    :value="nodes"
-    ref="slVueTree"
-    @select="makeActive"
-    @drop="handleSort"
-    @toggle="toggleFolder"
-    @contextmenu.native="showContextMenu()"
-    @nodecontextmenu="(node, event) => showContextMenu(node.data.id, event)"
-    @nodedblclick="node => sourceProperties(node.data.id)"
-    :scrollAreaHeight="50"
-    :maxScrollSpeed="15"
-  >
-
-    <template slot="title" slot-scope="{ node }">
-      <div class="title-container">
-        <span class="item-icon">
-          <i v-if="!node.isLeaf" class="fa fa-folder" ></i>
-          <i v-else-if="node.data.type === 'ffmpeg_source'" class="fa fa-film"></i>
-          <i v-else-if="node.data.type === 'image_source'" class="fa fa-image"></i>
-          <i v-else-if="node.data.type === 'slideshow'" class="fa fa-images"></i>
-          <i v-else-if="node.data.type === 'text_gdiplus'" class="fa fa-font"></i>
-          <i v-else-if="node.data.type === 'text_ft2_source'" class="fa fa-font"></i>
-          <i v-else-if="node.data.type === 'dshow_input'" class="fa fa-camera"></i>
-          <i v-else-if="node.data.type === 'wasapi_input_capture'" class="fa fa-microphone"></i>
-          <i v-else-if="node.data.type === 'wasapi_output_capture'" class="fa fa-volume-up"></i>
-          <i v-else-if="node.data.type === 'monitor_capture'" class="fa fa-desktop"></i>
-          <i v-else-if="node.data.type === 'game_capture'" class="fa fa-gamepad"></i>
-          <i v-else-if="node.data.type === 'browser_source'" class="fa fa-globe"></i>
-          <i v-else-if="node.data.type === 'scene'" class="fa fa-sitemap"></i>
-          <i v-else-if="node.data.type === 'color_source'" class="fa fa-paint-brush"></i>
-          <i v-else class="fa fa-file"></i>
+    <sl-vue-tree :value="nodes" ref="slVueTree" @select="makeActive" @drop="handleSort" @toggle="toggleFolder" @contextmenu.native="showContextMenu()" @nodecontextmenu="(node, event) => showContextMenu(node.data.id, event)" @nodedblclick="node => sourceProperties(node.data.id)" :scrollAreaHeight="50" :maxScrollSpeed="15">
```

Comment:
This is a really long line, can we keep the old formatting for this one?

Reply:
yeah agreed - I wonder if there's a way we can set Prettier not to change these